recipes/debreak/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.2" %}
+
+package:
+  name: debreak
+  version: {{ version }}
+
+source:
+    url: https://github.com/ChongLab/DeBreak/archive/refs/tags/debreak-v{{ version }}.tar.gz
+    sha256: 6162786bc60dc995d25790da6f806ed25bcfa50a7b10dc6e8404cacba7ce36df
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python <3
```

Comment:
Can we get a Python3 version of this tool? Python2 is EoL :(

Reply:
Thanks very much for reviewing and for the suggestion. The conda environment has been tested locally. The python3 version is planned to be updated. Could we release this build and upload future versions later? Thank you.